ci/jobs/macros.yaml

```diff
@@ -67,6 +67,16 @@
                 - semyers
                 - ttereshc
 
+# Ownership that defines owners for Bugzilla/Redmine integration job
+- property:
+    name: bz-redmine-ownership
+    properties:
+        - ownership:
+            owner: dadavis
```

Comment:
fyi, there are two accounts in Jenkins for @daviddavis There is dadavis and daviddavis. I'm not sure which is prefered. Maybe remove the one that isn't used?